docs/installing.md

```diff
@@ -1,131 +1,93 @@
-    ```
-
-    !!! admonition "See also"
 
-        Red Hat and CentOS with the yum package manager:
+**Next steps**: use the `percona-release` command to [setup the
+repository](repository-location.md) that contains the Percona product that you
+intend to install:
 
-        * [Percona Server for MySQL](https://www.percona.com/doc/percona-server/LATEST/installation/yum_repo.html)
-        * [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/LATEST/install/yum.html)
-        * [Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/install/yum.html)
-        * [Percona XtraBackup](https://www.percona.com/doc/percona-xtrabackup/LATEST/installation/yum_repo.html)
-        * [Percona Distribution for PostgreSQL](https://www.percona.com/doc/postgresql/LATEST/installing.html#on-red-hat-enterprise-linux-and-centos-using-yum)
-        * [Percona Distribution for MongoDB](https://www.percona.com/doc/percona-distribution-for-mongodb/LATEST/installation.html#install-on-red-hat-enterprise-linux-centos)
-        * [Percona Distribution for MySQL](https://www.percona.com/doc/percona-distribution-mysql/8.0/installing.html#install-pdmysql) 
+``` sh
+sudo percona-release setup <PRODUCT>
+```
+See [Configuring Percona Repositories with
+`percona-release`](percona-release.md) for additional information):
 
-> **Next steps**:  run the ``percona-release`` command to
-    [set up the repository](#usage) that contains the Percona product that you intend to install.
- 
-## Download and install percona-release from the website
+Next, use the operating system's package management tool to install the desired product package.
 
-1. On the [download page](https://www.percona.com/downloads/percona-release/), select the desired `percona-release` version from the _Version_ drop down.
-2. Download the `percona-release` package, relevant to your operating system. 
-3. Install `percona-release` using the command, relevant to your operating system:
+=== "Debian and Ubuntu with the `apt` package manager"
 
-=== "On Debian and Ubuntu"
+    * [Percona Distribution for MongoDB](https://www.percona.com/doc/percona-distribution-for-mongodb/LATEST/installation.html#install-on-debian-ubuntu)
+    * [Percona Distribution for MySQL](https://www.percona.com/doc/percona-distribution-mysql/8.0/installing.html#install-pdmysql)
+    * [Percona Distribution for PostgreSQL](https://www.percona.com/doc/postgresql/LATEST/installing.html#on-debian-and-ubuntu-using-apt)
+    * [Percona Server for MySQL](https://www.percona.com/doc/percona-server/LATEST/installation/apt_repo.html)
+    * [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/LATEST/install/apt.html)
+    * [Percona XtraBackup](https://www.percona.com/doc/percona-xtrabackup/LATEST/installation/apt_repo.html)
+    * [Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/install/apt.html)
```

Comment:
Some of these URLs need to be updated to the new locations, but I'll do that in a separate PR in order to avoid introducing more change.